app/javascript/components/mentoring/discussion/IterationHeader.tsx

```diff
@@ -10,6 +11,22 @@ export const IterationHeader = ({
   iteration: Iteration
   latest: boolean
 }): JSX.Element => {
+  const [testsStatus, setTestsStatus] = useState(iteration.testsStatus)
+
+  useEffect(() => {
+    const iterationChannel = new IterationChannel(
+      iteration.uuid,
+      (iteration) => {
+        console.log('boom')
```

Comment:
💥

Reply:
🧯

Reply:
@kntsoriano Can you add a `TODO` to these please to work out how we acutally deal with them later?

Reply:
Hahahahaha!